test/TestApp.AspNetCore.6.0/Controllers/ChildActivityController.cs

```diff
@@ -0,0 +1,46 @@
+// <copyright file="ChildActivityController.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using OpenTelemetry;
+
+namespace TestApp.AspNetCore._6._0.Controllers
```

Comment:
one day, we have to figure out how to prevent this code over and over.
we already have the "same" code for 3.1, 5.0, 6.0

Reply:
`*.tt` files? 🤣

https://docs.microsoft.com/visualstudio/modeling/code-generation-and-t4-text-templates